### PR TITLE
User side improved(review by ajax, company list, search company)

### PR DESCRIPTION
**Pull request by VatsalMewada**

### CODE REVIEW

File: app/models/company.rb [hunk 1]
```diff
@@ -26,4 +26,12 @@ class Company < ApplicationRecord
   def assign_default_role
     add_role(:company) if roles.blank?
   end
+
+  def self.search(search)
+    if search
+      Company.where("lower(name) LIKE ?", "%#{search.downcase}%")
```
Comment (swati-bacancy) [inline app/models/company.rb:32]: can you please use ilike

File: app/views/company_reviews/new.js.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+$('#new_review').html("<%= j render 'form', company_review: @company_review %>");
+$('#write_review').hide();
+$('#write_review2').hide();
```
Comment (swati-bacancy) [inline app/views/company_reviews/new.js.erb:3]:
```suggestion
$('#write_review2, #write_review').hide();
```
we can also hide multiple div so use this

File: app/views/company_reviews/new.js.erb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+$('#new_review').html("<%= j render 'form', company_review: @company_review %>");
```
Comment (swati-bacancy) [inline app/views/company_reviews/new.js.erb:1]: why are you passing this instance var??
no need to pass this

File: app/views/home/applied_job_list.html.erb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+                <div class="table-responsive">
+                  <table class="table table-hover">
+                    <thead class="thead-light">
+                      <tr>
+                        <th>Company</th>
+                        <th>Job Role</th>
+                        <th>Technology</th>
+                        <th>Experience</th>
+                        <th>CGPA</th>
+                        <th></th>
```
Comment (swati-bacancy) [inline app/views/home/applied_job_list.html.erb:22]: why extra TH?

File: app/views/home/index.html.erb [hunk 5]
```diff
@@ -1 +1,53 @@
-<h2>welcome to our site</h2>
+<div class="main-content">
+  <div class="section__content section__content--p30">
+    <div class="container-fluid">
+      <h2 align="center">Welcome <%= current_user.firstname %></h2>
+      <br>
+      <div class="col-lg-12">
+      <%= form_tag root_path, :method => 'get' do %>
```
Comment (swati-bacancy) [inline app/views/home/index.html.erb:7]: has new syntax

File: app/views/home/index.html.erb [hunk 6]
```diff
@@ -1 +1,53 @@
...
+              <table class='table table-hover'>
+                <thead>
+                  <tr>
+                    <th>Comapany list</th>
+                    <th>Avg. Rating</th>
+                    <th>No. of Reviewers</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <% if !@companies.blank? %>
```
Comment (swati-bacancy) [inline app/views/home/index.html.erb:25]: can you please use present

File: app/views/opening_jobs/index.html.erb [hunk 7]
```diff
@@ -1,30 +1,35 @@
...
+            <div class="table-responsive">
+              <%= link_to "Post a Job", new_company_opening_job_path , title: "Add new job" %></h6>
+              <table class="table table-hover">
+                <thead class="thead-light">
+                  <tr>
+                    <th>Job Role</th>
+                    <th>Technology</th>
+                    <th>Experience</th>
+                    <th>CGPA</th>
+                    <th></th>
```
Comment (swati-bacancy) [inline app/views/opening_jobs/index.html.erb:19]: why the extra th

Comment (VatsalMewada) [reply app/views/opening_jobs/index.html.erb:19]: For design purpose

File: app/views/home/applied_job_list.html.erb (hunk 4 above)
Comment (VatsalMewada) [reply app/views/home/applied_job_list.html.erb:22]: For design purpose

File: app/assets/stylesheets/common.css [hunk 8]
```diff
@@ -11,4 +11,10 @@
 .font-color-yellow{ color: yellow; }
 
 .container-padding{ padding-top: 10%;}
```
[the remark this answers is not in the document]
Comment (VatsalMewada) [reply app/assets/stylesheets/common.css:13]: **Can you please elaborate this?**

[the remark this answers is not in the document]
Comment (VatsalMewada) [reply app/assets/stylesheets/common.css:13]: > can you please change css class name like pt10pct
> 
> pt8pct

Done

**Pull request merged**